packages/opentelemetry-context-zone-peer-dep/package-lock.json

```diff
@@ -247,6 +247,11 @@
 			"integrity": "sha512-tsAQNx32a8CoFhjhijUIhI4kccIAgmGhy8LZMZgGfmXcpMbPRUqn5LWmgRttILi6yeGmBJd2xsPkFMs0PzgPCw==",
 			"dev": true
 		},
+		"@opentelemetry/context-base": {
```

Comment:
@dyladan I thought the package-lock.json files should not include the @opentelemetry/XXX packages linked by lerna

Reply:
Looks like dependabot got enabled somehow and doesn't understand lerna. This is a different bot than the one that has been keeping our dependencies updated.